compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     environment:
       <<: *common-variables
       LOGLEVEL: info
-      DEMO_MODE: False
+      DEMO_MODE: "False"
```

Comment:
For some reason, docker-compose wouldn't run when we didn't quote "False", with the quotes it was happy and ran.